### PR TITLE
remove all UI parts except swagger

### PR DESCRIPTION
**Pull request by wahyupratomoicsg**
#2 issue resolver => remove all UI parts except swagger, and make modification so swagger can be functional as API docs

### CODE REVIEW

File: openidm-zip/pom.xml [hunk 1]
```diff
@@ -168,17 +168,22 @@
             <version>${project.version}</version>
         </dependency>
 
+        <!--
```
Comment (tso-ic-sg) [inline openidm-zip/pom.xml]: @wahyupratomoicsg just remove them, dont comment off.

File: openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/MaintenanceService.java [hunk 2]
```diff
@@ -192,4 +192,8 @@ private void disableMaintenanceMode() {
         disableMaintenanceMode();
         return handleMaintenanceStatus();
     }
+
```
Comment (tso-ic-sg) [inline openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/MaintenanceService.java:195]: @wahyupratomoicsg got error if dont add this?

File: openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/UpdateLogServiceImpl.java [hunk 3]
```diff
@@ -138,4 +138,8 @@ public void updateUpdate(UpdateLogEntry entry) throws ResourceException {
                 new UpdateContext(ContextUtil.createInternalContext()),
                 Requests.newUpdateRequest(updateStore, entry.getId(), entry.toJson()));
     }
+    
+    protected void bindConnectionFactory(IDMConnectionFactory connectionFactory) {
```
Comment (tso-ic-sg) [inline openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/UpdateLogServiceImpl.java:142]: @wahyupratomoicsg got error if dont add this?

File: openidm-repo-orientdb/src/main/java/org/forgerock/openidm/repo/orientdb/impl/DBHelper.java [hunk 4]
```diff
@@ -432,6 +432,16 @@ private static void populateDefaultUsers(String defaultTableName, ODatabaseDocum
                 object(field(RelationshipUtil.REFERENCE_ID, "repo/internal/role/openidm-reg"))));
         populateDefaultUser(defaultTableName, db, anonymousUser, anonymousPwd, anonymousRoles);
         logger.trace("Created default user {} for registration purposes.", anonymousUser);
+
+        //create wahyu user
```
Comment (tso-ic-sg) [inline openidm-repo-orientdb/src/main/java/org/forgerock/openidm/repo/orientdb/impl/DBHelper.java]: @wahyupratomoicsg put some neutral test user name. dont put your name.

File: openidm-maintenance/src/test/java/org/forgerock/openidm/maintenance/upgrade/BundleHandlerTest.java [hunk 5]
```diff
@@ -142,131 +142,132 @@ private void startBundle(Bundle bundle) throws Exception {
...
-            FrameworkWiring frameworkWiring = service.getSystemBundle().adapt(FrameworkWiring.class);
-            frameworkWiring.refreshBundles(null);
-        } finally {
-            // cleanup this test
-            if (backupFile != null) {
-                Files.delete(backupFile);
-            }
-        }
-    }
+//    @Test
```
Comment (tso-ic-sg) [inline openidm-maintenance/src/test/java/org/forgerock/openidm/maintenance/upgrade/BundleHandlerTest.java:149]: @wahyufrombandung i think it is ok to comment off the failed tests first. i saw that there are comments inside. can put a top comment to state why we are commenting off when commenting off the test?

**Review by tso-ic-sg — CHANGES_REQUESTED** [review]
@wahyupratomoicsg okay some comments for changes

File: openidm-zip/pom.xml (hunk 1 above)
Comment (wahyupratomoicsg) [reply openidm-zip/pom.xml]: ok

File: openidm-repo-orientdb/src/main/java/org/forgerock/openidm/repo/orientdb/impl/DBHelper.java (hunk 4 above)
Comment (wahyupratomoicsg) [reply openidm-repo-orientdb/src/main/java/org/forgerock/openidm/repo/orientdb/impl/DBHelper.java]: ok

File: openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/UpdateLogServiceImpl.java (hunk 3 above)
Comment (wahyupratomoicsg) [reply openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/UpdateLogServiceImpl.java:142]: If I run "Build with dependencies" from netbeans, yeah they find this method

File: openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/MaintenanceService.java (hunk 2 above)
Comment (wahyupratomoicsg) [reply openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/MaintenanceService.java:195]: If I run "Build with dependencies" from netbeans, yeah they find this method

File: openidm-maintenance/src/test/java/org/forgerock/openidm/maintenance/upgrade/BundleHandlerTest.java (hunk 5 above)
Comment (wahyupratomoicsg) [reply openidm-maintenance/src/test/java/org/forgerock/openidm/maintenance/upgrade/BundleHandlerTest.java:149]: ok

**Comment by tso-ic-sg** [comment]
@wahyupratomoicsg okay Wahyu, sorry, can merge this into the "development" branch instead? close this pull request and create a new pull request to "development" branch.

**Comment by wahyu-pratomo** [comment]
done , had been closed and created a new pull request into development
branch

**Pull request closed without merging**